app/assets/stylesheets/common/base/header.scss

```diff
@@ -26,7 +26,7 @@
       .header-row {
         width: 100%;
 
-        .logo-wrapper {
+        .aspect-image-wrapper {
```

Comment:
This shouldn't be affected by this PR either (category logos have nothing to do with the header).